test/api/buffer/api_test.go

```diff
@@ -417,6 +412,10 @@ func RunRequests(
 		)
 	}
 
+	// Dump process stdout/stderr
+	assert.NoError(t, workingDirFs.WriteFile(filesystem.NewRawFile("process-stdout.txt", stdout.String())))
+	assert.NoError(t, workingDirFs.WriteFile(filesystem.NewRawFile("process-stderr.txt", stderr.String())))
+
```

Comment:
Vzdy sa dumpne stdout a stderr API servera v E2E teste, pre jednoduchsi debug.
Doteraz bolo nutne zavolat to znova s `TEST_VERBOSE` a dane hodnoty sa vypisali do konzoly.